ws-nextjs-app/pages/[service]/new_live/[id]/Posts/post.d.ts

```diff
@@ -0,0 +1,59 @@
+import { OptimoBlock } from '#models/types/optimo';
+
+interface Link {
+  url: string;
+  scheme: string;
+  host: string;
+  path: string;
+}
+
+interface Titles {
+  title: string | null;
+  source: string;
+}
+
+interface Descriptions {
+  text: string | null;
+  source: string;
+}
+
+interface Images {
+  url: string | null;
+  originalUrl: string | null;
+  altText: string | null;
+  copyright: string | null;
+  urlTemplate: string | null;
+}
+
+interface Page {
+  index: number;
+  total: number;
+}
+
+export interface Post {
+  typeCode?: string | null;
+  header: { model: { blocks: OptimoBlock[] } };
+  content: { model: { blocks: OptimoBlock[] } };
+  link: Link | null;
+  urn: string;
+  type: string;
+  options: {
+    isBreakingNews?: boolean;
+  };
+  dates: {
+    firstPublished: string;
+    lastPublished: string;
+    time?: string | null;
+    curated: string;
+  };
+  titles: Titles[];
+  descriptions: Descriptions[];
+  images: Images[];
+}
+
+export interface StreamResponse {
+  data: {
+    results: (Post | null)[];
```

Comment:
Yes this makes sense. And it explains why it through some issues up in our code! Since we took this interface from fabl-modules. I agree we should handle this in the BFF response.